src/lib/mail.ts

```diff
@@ -0,0 +1,45 @@
+import queryString from 'query-string';
+import {
+  isMobile,
+  osVersion,
+  osName,
+  fullBrowserVersion,
+  browserName,
+  deviceType,
+  mobileVendor,
+  mobileModel,
+} from 'react-device-detect';
+
+interface MailToParams {
+  bcc?: string;
+  cc?: string;
+  subject?: string;
+  body?: string;
+}
+
+const teamEmail = 'dreamscholars-contact@googlegroups.com';
+export function genMailToLink(params: MailToParams): string {
+  return `mailto:${teamEmail}?${queryString.stringify(params)}`;
+}
+
+const mobileInfo = `
+Device Type: ${deviceType}
+Vendor: ${mobileVendor}
+Model: ${mobileModel}`;
+
+export const bugReport = `
+---------- DO NOT EDIT THIS SECTION ----------
+OS: ${osName}
+OSVersion: ${osVersion} 
+Browser: ${browserName}
+Browser Version: ${fullBrowserVersion}
+${isMobile ? mobileInfo : ''}
+---------------------------------------------------------------
+Describe the bug:
+Steps to reproduce the behavior:
+Description of what you expected to happen:`;
```

Comment:
Couple observations:
1. It might difficult to spot this. On my Chrome browser on my Chrome OS laptop it opens up Gmail but the cursor appears at the beginning of the body text (does this match your experience?).
2. It might be easier to simply put this as one line distinctively separate from the rest of the template. E.g.

```
Describe the issue in detail, including steps to reproduce and expected behavior:



-----------------------------
...
-----------------------------
```

Reply:
1. yes the cursor appears at the beginning for me too